certified-connectors/StarRez REST v1/apiDefinition.swagger.json

```diff
@@ -7691,14 +7691,84 @@
                   ]
                 },
                 "ETA": {
-                  "type": "string",
-                  "description": "Minutes since midnight for estimated time of arrival.",
-                  "title": "ETA"
+                  "type": "object",
+                  "properties": {
+                    "Value": {
+                      "type": "string",
+                      "description": "Estimated time of arrival.",
+                      "title": "ETA",
+                      "format": "date-time"
+                    },
+                    "_operator": {
+                      "type": "string",
+                      "description": "The comparison operator to use.",
+                      "title": "ETA.Operator",
+                      "enum": [
+                        "GREATERTHAN",
+                        "LESSTHAN",
+                        "GREATERTHANOREQUALTO",
+                        "LESSTHANOREQUALTO"
+                      ],
+                      "x-ms-enum-values": [
+                        {
+                          "displayName": "Greater than",
+                          "value": "GREATERTHAN"
+                        },
+                        {
+                          "displayName": "Less than",
+                          "value": "LESSTHAN"
+                        },
+                        {
+                          "displayName": "Greater than or equal",
+                          "value": "GREATERTHANOREQUALTO"
+                        },
+                        {
+                          "displayName": "Less than or equal",
+                          "value": "LESSTHANOREQUALTO"
+                        }
+                      ]
+                    }
+                  }
                 },
                 "ETD": {
-                  "type": "string",
-                  "description": "Minutes since midnight for estimated time of departure.",
-                  "title": "ETD"
+                  "type": "object",
```

Comment:
The type has been changed for the input field. Existing flows will continue to send the field with the old type. ValuePath: operation/SelectTermSession/parameter/body/schema/ETD/type

Please confirm that the above change will not affect to the existing connection.

Reply:
I will revert this field back to string